server/models/RawCapture.js

```diff
@@ -79,92 +89,106 @@ const RawCaptureCreated = ({ id, lat, lon, field_user_id, field_username, attrib
     field_username,
     attributes,
     created_at,
-});
+  });
 
-const VerifyCaptureProcessed = ({ id, reference_id, type, approved, rejection_reason, created_at }) => Object.freeze({
+const VerifyCaptureProcessed = ({
+  id,
+  reference_id,
+  type,
+  approved,
+  rejection_reason,
+  created_at,
+}) =>
+  Object.freeze({
     id,
     reference_id,
     type,
     approved,
     rejection_reason,
-    created_at
-});
-
-const createRawCapture = (captureRepositoryImpl, eventRepositoryImpl) => (async (inputRawCapture) => {
-
-    // json wrap the 'attributes' array for storage in jsonb (storing array not suppported in jsonb)
-    const newRawCapture = {
-        ...inputRawCapture,
-        status: "unprocessed",
-        attributes: { entries: inputRawCapture.attributes },
-    };
-    const captureRepository = new Repository(captureRepositoryImpl);
-    const rawCapture = await captureRepository.add(newRawCapture);
-    const filteredAttr = rawCapture.attributes.entries
-        .filter(attribute => attribute.key === "app_flavor")
-    const rawCaptureCreated = RawCaptureCreated({
-        ...rawCapture,
-        attributes: filteredAttr
-    });
-    const raiseFieldDataEvent = raiseEvent(eventRepositoryImpl);
-    const domainEvent = await raiseFieldDataEvent(DomainEvent(rawCaptureCreated));
-    return { entity: rawCapture, raisedEvents: [domainEvent] };
-});
+    created_at,
+  });
 
+const createRawCapture = (captureRepositoryImpl, eventRepositoryImpl) => async (
+  inputRawCapture,
+) => {
+  // json wrap the 'attributes' array for storage in jsonb (storing array not suppported in jsonb)
+  const newRawCapture = {
+    ...inputRawCapture,
+    status: 'unprocessed',
+    attributes: { entries: inputRawCapture.attributes },
+  };
+  const captureRepository = new Repository(captureRepositoryImpl);
+  const rawCapture = await captureRepository.add(newRawCapture);
+  const filteredAttr = rawCapture.attributes.entries.filter(
+    (attribute) => attribute.key === 'app_flavor',
+  );
+  const rawCaptureCreated = RawCaptureCreated({
+    ...rawCapture,
+    attributes: filteredAttr,
+  });
+  const raiseFieldDataEvent = raiseEvent(eventRepositoryImpl);
+  const domainEvent = await raiseFieldDataEvent(DomainEvent(rawCaptureCreated));
+  return { entity: rawCapture, raisedEvents: [domainEvent] };
+};
 
 const FilterCriteria = ({
-    status = undefined,
-    field_username = undefined,
-    field_user_id = undefined
-}) => {
-    return Object.entries({ status, field_username, field_user_id })
-        .filter(entry => entry[1] !== undefined)
-        .reduce((result, item) => {
-            result[item[0]] = item[1];
-            return result;
-        }, {});
-}
-
-const QueryOptions = ({
-    limit = undefined,
-    offset = undefined,
+  status = undefined,

```

Comment:
I believe these are not real changes and some formatting triggered differences, am I right?